scripts/build.py

```diff
@@ -26,14 +26,14 @@ def install_with_ip(ip: str, is_dev: bool):
     if not is_dev:
       os.system('curl.exe -T ..\\plugin\\target\\aarch64-skyline-switch\\release\\libhdr.nro ftp://' + ip + ':5000/ultimate/mods/hdr-dev/plugin.nro')
     else:
-      os.system('curl.exe -T ..\\plugin\\target\\standalone\\aarch64-skyline-switch\\release\\libhdr.nro ftp://' + ip + ':5000/ultimate/mods/hdr-dev/plugin.nro')
```

Comment:
this is reordered so that the dev nro gets sent first (so you dont have to wait 15 seconds waiting for main plugin to get curled first lol